src/client/features/enrichment/index.js

```diff
@@ -6,9 +6,7 @@ const classNames = require('classnames');
 const queryString = require('query-string');
 
 const EnrichmentToolbar = require('./enrichment-toolbar');
-const EmptyNetwork = require('../../common/components/empty-network');
-const PcLogoLink = require('../../common/components/pc-logo-link');
-const CytoscapeNetwork = require('../../common/components/cytoscape-network');
+const { EmptyNetwork, PcLogoLink, CytoscapeNetwork } = require('../../common/components/');
```

Comment:
While we're here: I don't think this `EmptyNetwork` component is shown even if there is no content (i.e. nodes.length) unlike `features/interactions`.